Add an assessment for a DevOps Engineer (#5)
mysidewalk/interview: devops-engineer/README.md

# DevOps Engineer

## Prompt

Write a Dockerfile to create a docker image for the simple app contained in the `app` folder. Once you've built the image, please answer the following prompts: 
1. Run the container and verify that the app works. Briefly describe the process and the output.

> 2. Is your solution ready to be used in a production environment? Why?

Reviewer: ```suggestion
2. Is your solution ready for use in a production environment? Why?
```
Literally the only improvement hemingwayapp suggested when I ran it through the other day was the elimination of passive voice :shrug:

## Assessment Guidelines:
* Members of our engineering team will score submissions. They will be unaware of the author’s identity, resume, or background. They will have no access to information or documentation besides the submission.
* Scoring consists of 4 considerations of equal weight:
  * Runnability: was the assessor able to get the solution to run?
  * Functionality: does the solution solve the stated problem?
  * Readability: is the solution's code and documentation stylistically, idiomatically, and structurally sound? Does its structure, naming, and commenting make it easy to read?
  * Best Practice: does the solution follow best practices?

> * Scoring Functionality and Testing depends on the runnability of the submission.

Reviewer: ```suggestion
* Scoring Functionality depends on the runnability of the submission.
```
Removed "and Testing"
